dhis-2/dhis-api/src/main/java/org/hisp/dhis/userdatastore/UserDatastoreEntry.java

```diff
@@ -117,7 +117,7 @@ public void setKey( String key )
     @JsonProperty
     public String getValue()
     {
-        return encrypted ? encryptedValue : jbPlainValue;
+        return isEncrypted() ? getEncryptedValue() : getJbPlainValue();
```

Comment:
This is actually a bugfix. I think we can backport this entire PR to 2.38 and maybe even 2.37 as it is very local. If we decide not to backport we need to backport this line change as a fix.

Reply:
The issue here is that not using the getter can result in `null` even if the `value` field is non-null. This is because the getters return the `value` field with priority as long as it is non-null.